sdk/tests/conformance2/sync/sync-webgl-specific.html

```diff
@@ -77,11 +78,99 @@
   shouldBe("gl.getError()", "gl.NO_ERROR");
   gl.clientWaitSync(sync, 0, max + 1);
   shouldBe("gl.getError()", "gl.INVALID_OPERATION");
+
+  requestAnimationFrame(runGetSyncParameterTest);
+}
+
+function runGetSyncParameterTest() {
+  debug("");
+  debug("Verifying sync object isn't signaled too early");
+  sync = gl.fenceSync(gl.SYNC_GPU_COMMANDS_COMPLETE, 0);
+  // Verify as best as possible that the implementation doesn't allow a sync
+  // object to become signaled in the same frame, by spin-looping for some time.
+  var startTime = Date.now();
+  while (Date.now() - startTime < 2000) {
```

Comment:
300ms-1000ms is fine. 2000 seems unnecessarily high.

Reply:
Changed to 1000 ms.